client/gutenberg/extensions/slideshow/settings.js

```diff
@@ -0,0 +1,71 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+
+export const settings = {
+	name: 'slideshow',
+	prefix: 'jetpack',
+	title: __( 'Slideshow' ),
+	category: 'jetpack',
+	keywords: [ __( 'slideshow' ), __( 'image' ) ],
+	description: __( 'Add an interactive slideshow.' ),
+	attributes: {
+		align: {
+			type: 'string',
+		},
+		images: {
+			type: 'array',
+			default: [],
+			source: 'query',
+			selector: '.wp-block-slideshow_image_container',
+			query: {
+				alt: {
+					source: 'attribute',
+					selector: 'img',
+					attribute: 'alt',
+					default: '',
+				},
+				caption: {
+					type: 'string',
+					source: 'html',
+					selector: 'figcaption',
+				},
+				height: {
+					source: 'attribute',
+					selector: 'img',
+					attribute: 'data-height',
+				},
+				id: {
+					source: 'attribute',
+					selector: 'img',
+					attribute: 'data-id',
+				},
+				url: {
+					source: 'attribute',
+					selector: 'img',
+					attribute: 'src',
+				},
+				width: {
+					source: 'attribute',
+					selector: 'img',
+					attribute: 'data-width',
+				},
+			},
+		},
+		effect: {
+			type: 'string',
+			default: 'slide',
+		},
+	},
+	effectOptions: [
+		{ label: 'Slide', value: 'slide' },
```

Comment:
Should the labels be translated here?

Reply:
Good catch. Done in https://github.com/Automattic/wp-calypso/pull/29419/commits/ef53a1dcc0edc3be0a76d771099bf8d401e8516a